packages/dapp/src/components/Controlbar/Controlbar.tsx

```diff
@@ -12,11 +12,12 @@ import clsx from 'clsx';
 import { useTranslations } from 'next-intl';
 
 import ImportModal from '@/components/ImportModal';
-import DataStoreCombobox from '@/components/VCTable/DataStoreCombobox';
+// import DataStoreCombobox from '@/components/VCTable/DataStoreCombobox';
```

Comment:
Review imports and files such as `DataStoreCombobox` and `PlaygroundModal` and remove them completely if not in use anymore.